docs/testing/e2e-testing.md

```diff
@@ -201,6 +202,21 @@ For non-shadow content:
     });
 ```
 
+### Emulate a display
```

Comment:
Should we have a little copy here to kick things off? Just something like

> If you need to test how a component behaves with a particular viewport you can set the viewport width and height like so:

Reply:
Agreed ^